feathr_project/test/test_config_loading.py

```diff
@@ -6,31 +6,26 @@
 
 def test_configuration_loading():
     """
-    Test CLI init() is working properly.
+    Test the configuration can be overwritten by envs
     """
     runner = CliRunner()
     with runner.isolated_filesystem():
 
         result = runner.invoke(init, [])
 
```

Comment:
we should moving away from using CLI to test. We can just using the test_user_workspace here.

Reply:
agree